driver/web/docs/schemas/application/SurveyData.yaml

```diff
@@ -0,0 +1,85 @@
+type: object
+properties:
+  id:
+    type: string
+  key:
+    type: string
+  section:
+    type: string
+  allow_skip:
+    type: string
+  text:
+    type: array
+    items:
+      type: string
+  more_info:
+    type: string  
+  default_follow_up_key:
+    type: string
+  default_response_rule:
+    type: string      
+  follow_up_rule:
+    type: string
+  score_rule:
+    type: boolean
+  response:
+    type: object
+  type:
+    type: string
+  ok_answer:
+    type: object
+    nullable: true
+  ok_answers:
+    type: array
+    items: {}
+    nullable: true
+  options:
+    type: array
+    items: {}
+  nullable: true
+  action:
+    $ref: "./ActionData.yaml"
+    nullable: true
+  yes_no:
+    type: boolean
+    nullable: true
+  allow_multiple:
+    type: boolean
+    nullable: true
+  check_all:
+    type: boolean
+    nullable: true
+  ask_time:
+    type: boolean
+  minimum:
+      type: integer
+      format: float
+      nullable: true
+  maximum:
+      type: integer
+      format: float
+      nullable: true
+  whole_num:
+      type: boolean
+      nullable: true
+  slider:
+      type: boolean
+      nullable: true
+  self_score:
+      type: boolean
+      nullable: true
+  min_length:
+      type: integer
+      nullable: true
+  max_length:
+      type: integer
+      nullable: true
+  data_format:
+      type: object
+      nullable: true
+  body:
+      type: string
+      nullable: true
+  survey:
+      $ref: "./Survey.yaml"
+      nullable: true
```

Comment:
When using `$ref`, you cannot specify additional properties like `nullable` outside of the referenced doc. In this case, `nullable` can just be removed. Same applies to `action` above.